commodore/compile.py

```diff
@@ -115,7 +109,14 @@ def _local_setup(config, cluster_id):
             continue
         click.echo(f" > {c}")
         repo = git.init_repository(c)
-        config.register_component(c.name, repo)
+        component = Component(
+            name=c.name,
+            repo=repo,
+            version='master',
+            repo_url=None,
+            target_directory=P('dependencies') / c.name,
```

Comment:
I assume, that this will be the same for all instances. Consider adding some kind of a "constructor" that takes care of this.

Reply:
True. I'll create a property for this